src/parsers/manifest/dash/parse_adaptation_sets.ts

```diff
@@ -180,7 +191,11 @@ function getAdaptationID(
   if (isNonEmptyString(adaptation.attributes.frameRate)) {
     idString += `-${adaptation.attributes.frameRate}`;
   }
-  return idString;
+  if (idString.length === type.length) {
+    idString += representations.length > 0 ?
+      ("-" + representations[0].id) : "-empty";
+  }
+  return  (isTrickModeTrack ? "trickmode-" : "") + "adaptation-" + idString;
```

Comment:
It could have been better integrated in this function (like the same way than `isClosedCaption` and so on), no?

Reply:
Yes. I realise that we won't go into that logic if the adaptation has already an id. Do you think it is a problem? The trickmode track will not have any specific suffix.

I don't think it is problematic, as the track is supposed to have a distinct id, and will not be confounded in the API between other regular tracks.